code/game/machinery/deployable.dm

```diff
@@ -88,7 +88,7 @@
 	balloon_alert(user, "deconstructing barricade...")
 	if(!tool.use_tool(src, user, 2 SECONDS, volume=50))
```

Comment:
I think the fix would be to check for qdeleted here, since people can destroy the barricade while someone's crowbarring.

Reply:
use_tool already checks for qdeleted.